From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: 
 "chrome-official-brancher@chops-service-accounts.iam.gserviceaccount.com"
 <chrome-official-brancher@chops-service-accounts.iam.gserviceaccount.com>
Date: Tue, 28 Jul 2026 14:08:56 -0700
Subject: [PATCH] Roll commits from side projects

Roll src/third_party/skia from b4204f7ff931 to 14c2431eaed7
Commits rolled:
https://skia.googlesource.com/skia.git/+log/b4204f7ff931..14c2431eaed7

Generated by: http://go/bbid/8674999463047425313
Merge-Approval-Bypass: Chrome DEPS-autoroller

Change-Id: I025955e43aa9c486a05cd6a5b05eb5a2d104c2ae
Reviewed-on: https://chromium-review.googlesource.com/c/chromium/src/+/8160667
Bot-Commit: chrome-official-brancher@chops-service-accounts.iam.gserviceaccount.com <chrome-official-brancher@chops-service-accounts.iam.gserviceaccount.com>
Cr-Commit-Position: refs/branch-heads/7922_47@{#11}
Cr-Branched-From: c583827a354674180dc8856eb65aeb3f0afa5ee9-refs/branch-heads/7922@{#1740}
Cr-Branched-From: 059c884787b1bc15faa5afe5250f7d3a7e0ef64a-refs/heads/main@{#1654411}
---
 DEPS             | 2 +-
 third_party/skia | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/DEPS b/DEPS
index 60dbc44ab2d3a..e7beb34f33834 100644
--- a/DEPS
+++ b/DEPS
@@ -327,7 +327,7 @@ vars = {
   # Three lines of non-changing comments so that
   # the commit queue can handle CLs rolling Skia
   # and whatever else without interference from each other.
-  'skia_revision': 'b4204f7ff931ec662a690d00b2d1c19afeb3a4fe',
+  'skia_revision': '14c2431eaed7c0a097d197caa3669cda996c63b7',
   # Three lines of non-changing comments so that
   # the commit queue can handle CLs rolling V8
   # and whatever else without interference from each other.
diff --git a/third_party/skia b/third_party/skia
index b4204f7ff931e..14c2431eaed7c 160000
--- a/third_party/skia
+++ b/third_party/skia
@@ -1 +1 @@
-Subproject commit b4204f7ff931ec662a690d00b2d1c19afeb3a4fe
+Subproject commit 14c2431eaed7c0a097d197caa3669cda996c63b7
